training/metax/qwen1.5_MoE-megatron/config/config_C500x1x8.py

```diff
@@ -0,0 +1,10 @@
+batchsize = 1
+gbs = 512
+seqlength = 8192
+padlength = 8192
+precision = 'bf16'
+tensor_parallel = 1
+pipeline_parallel = 2
+accumulate_steps = 1
+theoryflops = 312000000000000.0
```

Comment:
should be Metax's TFLOPS or commit with "-1". Please also check your MFU results at README.md